apps/i18n/common/en_us.json

```diff
@@ -82,6 +82,8 @@
   "aiCannotAssess": "This learning goal is too subjective for AI to evaluate.",
   "aiConfidence": "AI has {aiConfidence} confidence in this assessment",
   "aiConfidenceTooltip": "The confidence score is calculated from how well the AI assessment worked for this learning goal on our training set of student projects. Pay extra attention to projects with medium or low confidence.",
+  "aiEvaluation_student_must_submit": "Student has not submitted project for evaluation. Run AI evaluation from the settings tab.",
```

Comment:
the ui string name suggests the student needs to submit, but the string value says the teacher needs to press the run button. reading how you are using this string, it seems like the string value should say that the student needs to submit the project, since the AI cannot evaluate if the student has done no work.